src/samples/digest/v1/minimum.json

```diff
@@ -0,0 +1,42 @@
+{
+    "id": "1",
```

Comment:
For the ``id`` value, my suggestion is something like ``"digest-99999"``, the ``99999`` being the article the digest is associated with. There are no constraints for me on this value, so please suggest an alternate if you don't like it.

Reply:
I wouldn't include the type in the ID, and would be wary of using an article ID as it's a `1..n` relationship not `1..1`. But that's not important for now as it's just a string.